README.md

```diff
@@ -19,6 +19,21 @@ dependencies {
 }
 ```
 
+```kotlin
```

Comment:
"Kotlin" could be mentioned outside the code block so that it is descriptive when rendered. Here's rich diff:
https://github.com/willowtreeapps/assertk/pull/401/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5